packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -6,10 +6,26 @@
  */
 import Cookies from 'js-cookie'
 
-export interface BaseStorage {
-    set(key: string, value: string, options?: unknown): void
-    get(key: string): string
-    delete(key: string): void
+export type StorageTypes = 'cookie' | 'local'
```

Comment:
```suggestion
export type StorageTypes = 'cookie' | 'local' | 'memory'
```
(Assuming we use the suggested `MemoryStorage` name.)

Even if we don't use `memory` ourselves, would adding this make sense?

Reply:
Nice catch. 🙇